apps/events-helsinki/browser-tests/search-page/search-page.testcafe.ts

```diff
@@ -0,0 +1,25 @@
+import { initTestI18n as i18n } from '../../../../packages/common-i18n/src/';
+import {
+  EventSearchPage,
+  getEnvUrl,
+} from '../../../../packages/common-tests/browser-tests';
+import { ROUTES } from '../../src/constants';
+import EventsConsentModal from '../page-models/events-consent-modal';
+
+fixture('Search page')
+  .page(getEnvUrl(ROUTES.SEARCH))
+  .beforeEach(async () => i18n.changeLanguage('default'));
+
+// TODO: Load this from fixture or something
+// so that it does not need to be repeated in every test
+const acceptAllCookies = async () => {
```

Comment:
You are talking about the testcafe user roles: https://testcafe.io/documentation/402845/guides/advanced-guides/authentication#user-roles
It's not heavy to refactor, so feel free to do a cookie consent accepter role :)

Reply:
Actually it probably works quite fast:
> Faster test execution. TestCafe only executes the log-in routine once in each of the browsers you open. During subsequent Role activations, TestCafe re-uses Role authentication data (cookies and other artifacts) to [instantly](https://testcafe.io/documentation/402845/guides/advanced-guides/authentication#how-roles-work) log users in and out.

So I would assume it only accepts the cookies in the first test. Then it autoset the created cookies so the consent modal wont even pop up for the next tests. 

> While the Role is active, TestCafe saves the authentication data from cookie storage, localStorage, and sessionStorage to the active Role object. If your web application stores authentication data elsewhere, TestCafe cannot save it.

Reply:
I added one! Please, review :)